src/redux/reducers.js

```diff
@@ -673,6 +774,20 @@ export function posts(state = {}, action) {
     case response(ActionTypes.GET_SINGLE_POST): {
       return updatePostData(state, action)
     }
+    case ActionTypes.REALTIME_POST_NEW:
+    case ActionTypes.REALTIME_POST_UPDATE: {
+      return { ...state, [action.post.id]: postParser(action.post) }
+    }
+    case ActionTypes.REALTIME_COMMENT_NEW: {
+      const post = state[action.comment.postId]
+      if (!post || post.comments.indexOf(action.comment.id)!==-1) {
```

Comment:
For now I'm patching in here and there, but obviously need to make some kind on `postMapper` to prevent this type of issues. Ideally we'll call it on each API response, but this needs to be properly designed.